docs/source/05_data/01_data_catalog.md

```diff
@@ -260,6 +261,37 @@ us_corn_yield_data:
 
 > *Note:* When using [`pandas.SQLTableDataSet`](/kedro.extras.datasets.pandas.SQLTableDataSet) or [`pandas.SQLQueryDataSet`](/kedro.extras.datasets.pandas.SQLQueryDataSet) you must provide a database connection string. In the example above we pass it using `scooters_credentials` key from the credentials (see the details in [Feeding in credentials](#feeding-in-credentials) section below). `scooters_credentials` must have a top-level key `con` containing [SQLAlchemy compatible](https://docs.sqlalchemy.org/en/13/core/engines.html#database-urls) connection string. As an alternative to credentials, you could explicitly put `con` into `load_args` and `save_args` (`pandas.SQLTableDataSet` only).
 
+Example 14: Loading data from Minio (S3 API Compatible Storage)
```

Comment:
I would like to make this example as concise as possible, like other examples. How about

Example 14: Loading data from [Minio](https://min.io/) (S3 API Compatible Storage)
```yaml
test:
  type: pandas.CSVDataSet
  filepath: s3://your_bucket/test.csv # assume `test.csv` is uploaded to the Minio server.
  credentials: dev_minio
```

In `credentials.yml`, define the access key, token and the endpoint_url as follows:
```yaml
dev_minio:
  key: token
  secret: key
  cleitn_kwargs:
    endpoint_url : 'http://localhost:9000' 
```
> Note: The easiest way to setup Minio is to run a docker. After the following command, you can access to Minio server with http://localhost:9000 and create a bucket and add files as if it is on S3.
```bash
docker run -p 9000:9000 -e "MINIO_ACCESS_KEY=token" -e "MINIO_SECRET_KEY=key" minio/minio server /data
```

Reply:
looks better, updated